fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+                  
+        '404':
+          description: Not Found 
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'          
+    
+        '422':
+          description: Unprocessable Entity
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'    
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X PUT -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/#{id}/complete.json?payment_gateway_id=null&payment_pending=false" 
+            
+        - lang: Ruby
+          source: |
+            require 'shopify_api'
+  
+            shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+            ShopifyAPI::Base.site = shop_url
+            draft_order = ShopifyAPI::DraftOrder.find(#{id})
+            draft_order.send_invoice     
+            
+            
+  '/admin/draft_orders/#{id}/checkout.json':
+    post:
+      summary: Create or retrieve existing checkout
+      description: >-
+        This call will either retrieve an existing checkout for this draft order or will create a new checkout if none exists previosuly.  
+        
+        ![To Be Implemented](https://andrewjtech123.github.io/imagehost/tag-implemented.png) 
+        
+        
+      
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: integer
+      responses:
+        '201':
+          description: Checkout object created or retrieved.
+          schema:
+            $ref: '#/definitions/checkout'
+            
+        '400':
+          description: Bad Request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                  
+            
+            
+      
+      tags:
+        - Create
+        - To Be Implemented
+        
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders/#{id}/checkout.json
+            
+        - lang: Ruby
+          source: |
+            require 'shopify_api'
+
+            shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+            ShopifyAPI::Base.site = shop_url
+            draft_order = ShopifyAPI::DraftOrder.find(#{id})
+            draft_order.checkout    
+            
+definitions:
+  DraftOrder:
+    type: object
+    properties:
+      id:
+        type: string
+        description: The `id` of the draft order.
+        readOnly: true
+      order_id:
+        type: string
+        description: The `id` of the order associated to the draft order, once created.
```

Comment:
* perhaps "associated to" -> "associated with"
* "once created" -> "after it's been created"